environment.yml

```diff
@@ -39,7 +39,7 @@ dependencies:
   - xarray  # Can be installed from PyPI, but here to get a consistent set of depencies with iris.
   # Non-Python dependencies
   - graphviz
-  - cdo=1.9.3  # see notes for future version
+  - cdo=1.9.5  # see notes for future version
 
```

Comment:
nooo dude, we're back in the bogs with this - it may work for you but on Jasmin it can not resolve the environment with 1.9.5....

Reply:
The tests failed on CircleCI for 1.9.3 (see above or https://circleci.com/gh/ESMValGroup/ESMValTool/15748?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link), so I thought let's try something else and see if that works.

Reply:
I'm going home now, feel free to try other versions ;-)

Reply:
ie it spins and spins and spins around

Reply:
I know, I've been looking at it all day! Let's see what happens if we give it another hint.

Reply:
:hammer: :hammer: :hammer: :grin: 

yeah I'll try more versions after trying them for almost 2 days now, love you, bud hahah
No seriously, it might have worked for you since you've probably done an env update but the env build from scratch via `env create` is `Solving environment \` untill Brexit, not working. Why are the tests failing with 1.9.3? They work for me fine (ie cdo --version), I bet you had to downgrade hdf5 as I mentioned and you didn't. Anyways, good weekend, man!

Reply:
I...think...it works, man...BLOODY HELL!! Yes dude, managed to resolve the env on both Jasmin and my laptop, create it, install the tool, run the tests (not the installation ones), basic commands in esmvaltool, ncl and cdo and even the CI had a brief moment where only the python3_install had failed (check out that cdo copy - the command works but there is something wrong with that tmp.nc file). :beer:  :beer: :beer:

Reply:
well nothing wrong with that tmp file, it's just cdo is screwing things up again - here is the cause for the issue: https://github.com/koldunovn/nk_public_notebooks/blob/master/Install%20climate%20data%20operators%20%28cdo%29%20on%20Ubuntu%20with%20netCDF4%20and%20hdf5%20support.ipynb from our own Nikolay

Reply:
cdo is a stinking pile of crap, here's another pointer that may or may not be useful: https://code.mpimet.mpg.de/boards/1/topics/1445 - we do have `NetCDF library version : 4.6.1 of Nov 15 2018 18:26:07 $` so it should work

Reply:
I'll try to come up with a working test

Reply:
no, this is a valid test if we use cdo with netCDF4 files (do we?) - it's just that the bloody thing needs to be compiled with netcdf libraries; if we don't use netcdf4 with cdo then you can just change the test to a csv or txt file, but I doubt that will be useful; I'll see if updating the  pypi netcdf4 to 1.4.3.2 (current version) will fix it

Reply:
I think the test failed because cdo expects certain coordinates to be present in the netcdf file. I changed it so it works (or fails if the environment is broken).

Reply:
yeah just found this very thing myself on some cdo forum; the error message was misleading to say the least pfff